HIP/hip-227.md

```diff
@@ -71,7 +74,6 @@ Optional filters
 - `limit`
 - `order`
 - `timestamp`
-- `address`
 - `topic0`
```

Comment:
We should mention able the idiosyncrasies around this API including that timestamp is required if a topic is supplied. Also that timestamp must have a to and from within 7 days of each other. And that there's no paging so users will need to adjust their parameters on their own if more than page size results.

Reply:
Noted, quite a few for this endpoint